diff -up libcgroup-3.1.0/samples/config/cgconfig.conf.config-patch libcgroup-3.1.0/samples/config/cgconfig.conf
--- libcgroup-3.1.0/samples/config/cgconfig.conf.config-patch	2023-10-17 15:40:32.792681299 +0200
+++ libcgroup-3.1.0/samples/config/cgconfig.conf	2023-10-17 15:41:18.629062142 +0200
@@ -1,42 +1,4 @@
-# SPDX-License-Identifier: LGPL-2.1-only
-#
-#  Copyright IBM Corporation. 2007
-#
-#  Authors:	Balbir Singh <balbir@linux.vnet.ibm.com>
-#
-#group daemons/www {
-#	perm {
-#		task {
-#			uid = root;
-#			gid = webmaster;
-#		}
-#		admin {
-#			uid = root;
-#			gid = root;
-#		}
-#	}
-#	cpu {
-#		cpu.shares = 1000;
-#	}
-#}
-#
-#group daemons/ftp {
-#	perm {
-#		task {
-#			uid = root;
-#			gid = ftpmaster;
-#		}
-#		admin {
-#			uid = root;
-#			gid = root;
-#		}
-#	}
-#	cpu {
-#		cpu.shares = 500;
-#	}
-#}
-#
-#mount {
-#	cpu = /mnt/cgroups/cpu;
-#	cpuacct = /mnt/cgroups/cpuacct;
-#}
+# By default, we expect systemd mounts everything on boot,
+# so there is not much to do.
+# See man cgconfig.conf for further details, how to create groups
+# on system boot using this file.
diff -up libcgroup-3.1.0/samples/config/cgconfig.sysconfig.config-patch libcgroup-3.1.0/samples/config/cgconfig.sysconfig
--- libcgroup-3.1.0/samples/config/cgconfig.sysconfig.config-patch	2023-07-28 22:12:30.000000000 +0200
+++ libcgroup-3.1.0/samples/config/cgconfig.sysconfig	2023-10-17 15:40:32.792681299 +0200
@@ -5,8 +5,6 @@
 # controller to limit cpu.shares of this default group and allowing some more
 # important group take most of the CPU.
 #
-# By default, create these groups:
-CREATE_DEFAULT=yes
-
-# Uncomment following line to disable creation of the default group on startup:
-# CREATE_DEFAULT=no
+# By default, do not create these groups:
+# CREATE_DEFAULT=yes
+CREATE_DEFAULT=no
