diff -up libcgroup-3.1.0/src/api.c.5~ libcgroup-3.1.0/src/api.c
--- libcgroup-3.1.0/src/api.c.5~	2023-10-17 15:50:43.678694567 +0200
+++ libcgroup-3.1.0/src/api.c	2023-10-17 15:51:37.250129865 +0200
@@ -4514,9 +4514,9 @@ int cgroup_change_cgroup_flags(uid_t uid
 					written = snprintf(newdest + j,	available, "%d", pid);
 					break;
 				case 'p':
-					if (procname) {
+					if (procname && strlen(basename(procname))) {
 						written = snprintf(newdest + j,	available, "%s",
-								   procname);
+								   basename(procname));
 					} else {
 						written = snprintf(newdest + j,	available, "%d",
 								   pid);
